Cargo.toml

```diff
@@ -91,7 +91,7 @@ members = [
     "secure/storage",
     "secure/storage/github",
     "secure/storage/vault",
-    "sf-stream",
+    "fh-stream",
```

Comment:
Lint failed. This needs to be alphabetically ordered I think.

Reply:
Fixed and push, let's see now :)